src/app/shared/models/icon.ts

```diff
@@ -4,9 +4,10 @@ export enum Icon {
     Bold = 2,
     Trezor = 3,
     Unchained = 4,
-    Purple = 5,
-    Teal = 6,
-    Rust = 7,
+    Blockstream = 5,
+    Purple = 6,
```

Comment:
Wallet Icons are saved using this integer. By adjusting the value for Purple, teal and rust when a user updates to a new version of Branta core the icon will change.

Example: If I run Branta 0.6.0 and select Purple (5) then run this code I will see Blockstream (5).

I think having Blockstream above the color options is correct, that order is determined in the IconOptions variable.